### PR TITLE
Add sandbox, license, banner

### PR DESCRIPTION
**Pull request by ishanmkh**
- Add sandbox setup in install to avoid folks w/o K8s churning
- Merged running scripts into demo
- Update banner for community beta
- Move assets into a folder of optics and org
- Other minor updates

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,97 +18,77 @@ We're building up Pixie for broad use by the end of 2020 with an initial focus o
...
-2. **Run Scripts**
+**Run scripts with `px` CLI**
 
-    Service SLA `px run px/service_stats`
+Access data in seconds by running pre-built community scripts like:
 
-    Node health: `px run px/node_stats`
+- Service SLA: `px run px/service_stats`
+- Node health: `px run px/node_stats`
+- MySQL metrics: `px run px/node_stats`
```
Comment (zasgar) [inline README.md:69]: should be mysql_stats

File: README.md [hunk 2]
```diff
@@ -18,97 +18,77 @@ We're building up Pixie for broad use by the end of 2020 with an initial focus o
...
 
-1. **Create account**
+1. **Signup**: Visit our [product page](https://work.withpixie.ai/) and signup with your google account.
 
-    Visit our [product page](https://work.withpixie.ai/) and signup with your google account.
-
-2. **Install**
-
-    Copy and run this command to install the CLI in your environment.
+2. **Install CLI:** Copy and run this command to install `px` in your environment
```
Comment (zasgar) [inline README.md:39]: It's not clear what px is. It's guessable by the context but not super clear. Can consider rewording.

File: README.md [hunk 3]
```diff
@@ -18,97 +18,77 @@ We're building up Pixie for broad use by the end of 2020 with an initial focus o
...
-    MySQL metrics: `px run px/node_stats`
+![CLI Demo](./pxl_scripts/http_data.svg)
 
-    Explore more scripts by running `px scripts list` or by hitting cmd+k in the [Live UI](https://work.withpixie.ai/) 
-    
-    Check out our [pxl_scripts](pxl_scripts) repo for more examples.
+**View results on machine generated Live dashboards**
 
-3. **View and Itegrate Results**
+Pixie Platform auto-generates "Live View" dashboard to visualize script results. You can view them by clicking on the URLs prompted by `px` or by visiting: `https://work.withpixie.ai/live`
```
Comment (zasgar) [inline README.md:82]: either "The Pixie Platform" or Pixie's Platform

**Comment by ishanmkh** [comment]
thanks @zasgar ! updated.

**Pull request merged**